vendors/circom/prover_main.cc

```diff
@@ -269,6 +275,11 @@ int RealMain(int argc, char** argv) {
     tachyon_cerr << "num_runs should be positive" << std::endl;
     return 1;
   }
+
+  base::Profiler profiler({.output_path = trace_path});
```

Comment:
This is a c++20 feature See https://en.cppreference.com/w/cpp/language/aggregate_initialization